### PR TITLE
Replaced 14gqr and 16fil with 1 supernova and 1 varstar from fritz's sources list

### PR DESCRIPTION
**Pull request by bparazin**
To fix issue #2089, I replaced 14gqr and 16fil with https://fritz.science/source/ZTF21aaqjmps and https://fritz.science/source/ZTF18aabcvnq. Those two were chosen since they are a supernova & a var star to hit 2 object types. This is my first actual pull request that isn't just editing a markdown file, so apologies if I've messed anything up!

### CODE REVIEW

**Comment by dmitryduev** [comment]
Thanks @bparazin and congrats on your first PR! Would you mind posting some screenshots?

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@rjsf/core": "2.5.1",
     "@rjsf/material-ui": "2.4.0",
     "autosuggest-highlight": "3.1.1",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
```
Comment (stefanv) [inline package.json:19]:
```suggestion
    "check-dependencies": "1.1.0",
```

**Comment by stefanv** [comment]
Thank you for your first PR @bparazin!

To pass the linter, you may need to edit the `packages.json` file as suggested.

The most likely pushback on this PR is that it removes our favorite comment: "birds are not real" :D

**Comment by bparazin** [comment]
image
image
image
image
image

Haha, I can add that comment back :sweat_smile:

File: data/ZTF21aaqjmps_phot.csv [hunk 2]
```diff
@@ -0,0 +1,203 @@
+filter,mjd,flux,zp,magsys,fluxerr
```
Comment (dmitryduev) [inline data/ZTF21aaqjmps_phot.csv:1]: i-band data (ztfi) is probably still private, so please remove it from the commit.

**Review by acrellin — COMMENTED** [review]
Looks good overall, and thanks for your first PR! 🎉 
Couple of minor comments below

File: data/db_demo.yaml [hunk 3]
```diff
@@ -530,60 +455,31 @@ observing_run:
...
-    comment: Please integrate for 1200s
-  - obj_id: 16fil
+    comment: Please get  3x100s
+    status: "complete"
+  - obj_id: ZTF18aabcvnq
     run_id: =esi_run
-    priority: "2"
-    comment: Please take only when seeing is less than 1.5"
+    priority: "3"
+    comment: Please only take when seeing is below 2
```
[the remark this answers is not in the document]
Comment (stefanv) [reply data/db_demo.yaml:466]: Hrm, quotes needed, not need? Probably 2!

File: data/db_demo.yaml [hunk 4]
```diff
@@ -415,84 +396,28 @@ sources:
...
-  - obj_id: 16fil_unsaved_copy
-    origin: "Program A"
-    data:
-      numeric_field: 0.98765
-      some_boolean: false
-    group_ids:
-      - =program_A
-  - obj_id: 16fil
-    origin: "Program A"
+      some_text: "Some auto-annotated text"
```
Comment (stefanv) [inline data/db_demo.yaml:414]: What is realistic annotated text?

File: data/db_demo.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (acrellin) [reply data/db_demo.yaml:466]: The " is the units here

[the remark this answers is not in the document]
Comment (stefanv) [reply data/db_demo.yaml:466]: Ah! What's yaml going to do with this?  Perhaps should be `\"` or quoted with single quotes?

**Comment by acrellin** [comment]
@bparazin it looks like the data loader is failing

**Comment by bparazin** [comment]
> @bparazin it looks like the data loader is failing

I change 2 lines in the yaml and it goes crazy 😅. Let me take a look at this

**Comment by bparazin** [comment]
Are these failed checks ones I have to worry about? Because for the life of me I cannot figure out where they've come from

**Comment by profjsb** [comment]
Happy to approve one merge conflicts have been resolved.

**Comment by bparazin** [comment]
Dang, this comment change has made this certainly more tricky than expected; will fix loader issues in a minute

**Comment by bparazin** [comment]
@profjsb, @acrellin, @stefanv Any thoughts on the commits I referenced above?

**Review by acrellin — APPROVED** [review]
Looks good to me! Is there no publicly available spectrum for ZTF18aabcvnq?

**Comment by bparazin** [comment]
@acrellin I don't see any on [here](https://fritz.science/source/ZTF18aabcvnq), is there somewhere else it would be?

**Comment by dmitryduev** [comment]
🥳

**Pull request merged**